share/functions/help.fish

```diff
@@ -78,6 +78,11 @@ function help --description 'Show help for the fish shell'
 		return 1
 	end
 
+	# In Cygwin, start the user-specified browser using cygstart
+	if begin; type -q cygstart; and test $fish_browser != "cygstart"; end
+		set fish_browser cygstart \"$fish_browser\"
+	end
+
```

Comment:
There are some cases in which due to fish lacking `&&` and `||` for conditional statements one has no choice but to do something like that, but in this case the conversion is possible, so I vote for the most readable one.